net/wireguard/src/opnsense/scripts/Wireguard/wg-service-control.php

```diff
@@ -68,7 +68,10 @@ function wg_start($server, $fhandle, $ifcfgflag = 'up')
     }
     mwexecf('/usr/bin/wg setconf %s %s', [$server->interface, $server->cnfFilename]);
 
-    foreach (explode(',', (string)$server->tunneladdress) as $alias) {
+    /**
+     * The tunneladdress can be empty, so any string with a length of 0 bytes is removed.
+     */
+    foreach (array_filter(explode(',', (string)$server->tunneladdress), 'strlen') as $alias) {
```

Comment:
much better, thanks. I'm pondering a bit about the strlen here and https://www.php.net/manual/en/function.array-filter.php says:

If no callback is supplied, all empty entries of array will be removed. See [empty()](https://www.php.net/manual/en/function.empty.php) for how PHP defines empty in this case.

Reply:
Ah yes it looks like strlen shouldn't be even needed when no callback has the same result. I will test this right away.